docs/src/components/component/icon-square/icon-square.html

```diff
@@ -0,0 +1,6 @@
+
+<svg class="icon-{{ iconSize }} icon-fill {% if isBorder %}icon-bordered {% endif %}{% if iconStatus %}icon-{{ iconStatus }}{% endif %}">
```

Comment:
Yea we weren't able to convert them, maybe it should be some future goal, but I'm not sure it's possible.